src/TextUI/Command.php

```diff
@@ -662,7 +675,6 @@ protected function handleArguments(array $argv): void
                     break;
 
                 case '--version':
-                    $this->printVersionString();
```

Comment:
I would expect the version string to be printed, but interestingly, it doesn't. Perhaps because the printer uses output buffering?

Reply:
Maybe it makes sense to flush it when the object is destructed?